packaging/alvr_build_linux.sh

```diff
@@ -26,11 +26,18 @@ rawContentProvider='https://raw.githubusercontent.com'
 # Android NDK version
 ndkVersion=30
 
-# Grab the directory git creates
-repoDir="$(dirname "$(realpath "${0}")")/$(basename "${repo}")"
+# Grab the repository directory
+maybeRepo="$(dirname "${0}")/../"
+if [ -d "${maybeRepo}/.git" ]; then
+    repoDir="${maybeRepo}"
+else
+    repoDir="$(dirname "$(realpath "${0}")")/$(basename "${repo}")"
```

Comment:
This took me forever to understand. Add a comment.

Reply:
Yea this is better, fixed.